plugin/ctrlsf.vim

```diff
@@ -139,6 +139,7 @@ let s:default_mapping = {
     \ "open"    : ["<CR>", "o"],
     \ "openb"   : "O",
     \ "split"   : "<C-O>",
+    \ "vsplit"  : "<C-o>",
```

Comment:
vim can't distinguish `<C-O>` from `<C-o>`, they share same character code. I think just leave map for vsplit empty should be ok.